irohad/ametsuchi/temporary_wsv.hpp

```diff
@@ -51,7 +51,7 @@ namespace iroha {
        */
       virtual bool apply(const model::Transaction &transaction,
                          std::function<bool(const model::Transaction &,
-                                            CommandExecutor &, WsvQuery &)>
+                                            WsvCommand &, WsvQuery &)>
                              function) = 0;
```

Comment:
Ah, I see. I mistake.
Is `WsvQuery` in `validate()` `TemporaryWsv`?
And is `WsvQuery` in `execute()` `MutableStorage`?

Sorry, I have a more question.
Which `validate()` and `Context` check the permission to execute the command?